README.md

```diff
@@ -90,7 +92,7 @@ class InvoicePaid extends Notification
 }
 ```
 
-Here's a screenshot preview of the above notification on Facebook Messenger:
+The notification is being sent to the Facebook page you have created before and you will find the message inside the chat window. Here's a screenshot of how this looks:
```

Comment:
It's a bit confusing. It sounds like we're sending a notification to a page and not to the user.